### DISCUSSION TITLE
Measure one-way time of flight using perpendicular-to-path Sunlight to start clock?

### DISCUSSION THREAD

**Discussion by sahil5d** [a-cosmology-group/acg#154] [Astrophysics and Physics]
There may be yet another way to measure the one-way time of flight of a photon.

Let's say there's a test path AB, 1km long. There's an emitter at A, and a clock at B.

The challenge has been to emit the photon at A and start the clock at B, at the same time. What if this can be solved by a distant light source, very distant, and perpendicular to the path AB? What if Sunlight can be the trigger for the the emitter at A, and the clock at B?

I think so. And this can be done at sunrise (or sunset) and triggered by photosensors at A and B if the ambient light is above (or below) some threshold.

Given `v_abs` estimated at c/1000, a nanosecond-precision clock is adequate.
If AB is in the direction of `v_abs`, t would ≈ 3339 ns.
If AB is against `v_abs`, t ≈ 3332 ns.
If AB is perp to `v_abs`, t ≈ 3336 ns.

What does ACG think?

This might also be achievable with a trigger signal from a geosynchronous satellite (1km / 36,000km << 1/1000).

**Comment by bgreenman** [comment]
Lets try the experiment see what happens to measure the photon's flight

**Comment by sahil5d** [comment]
I might be wrong about this approach. Setting up the path AB to be perpendicular to the distant light is not guaranteeable, because of aberration.

**Reply by bgreenman** [reply]
correct, no experiment's result is guaranteed.

**Reply by RedshiftDrift** [reply]
The problem is, even if you use a very distant trigger $T$ (like a millisecond pulsar), the paths $\overline{TA}$ and $\overline{TB}$ cannot be both perpendicular to $\overline{AB}$ because in every case $ATB$ is a triangle by design.
The velocity component of the trigger pulse parallel to $\overline{AB}$ makes the configuration equivalent to [a two-way light synchronization (see first image)](https://sites.pitt.edu/~jdnorton/Goodies/rel_of_sim/index.html).

**Reply by sahil5d** [reply]
That's not the problem. The trigger (like a star) would be so wide that the photons wouldn't be from one point T. The path of the photon to A and the path of the photon to B would be parallel.

The problem is ensuring AB to be perpendicular to the distant light. I had thought.. use a telescope at A, get it aligned to the distant light, then draw a path perpendicular away from the end of the telescope tube and that be AB. But the telescope will be tilted forward by θ=v/c (aberration), so the constructed path AB will be tilted down by the same. So AB will not actually be perpendicular to the distant light, and the AB tilt will exactly cancel out the expected one-way time of flight difference.

**Reply by bgreenman** [reply]
correct
